tests/h/traversal/roots_test.py

```diff
@@ -382,10 +367,12 @@ def groups(self, factories):
         return [factories.Group(), factories.Group(), factories.Group()]
 
     @pytest.fixture
-    def group_service(self, pyramid_config):
-        group_service = mock.create_autospec(GroupService, spec_set=True, instance=True)
-        pyramid_config.register_service(group_service, name="group")
-        return group_service
+    def groupfinder_service(self, pyramid_config):
+        groupfinder_service = mock.create_autospec(
+            GroupService, spec_set=True, instance=True
+        )
+        pyramid_config.register_service(groupfinder_service, name="group")
+        return groupfinder_service
```

Comment:
This mock is for `GroupService` rather than `GroupfinderService`.

Reply:
Good spot. This was one of the awkward files which referred to both as "group_service" so some automated renaming here has caused trouble.

Added a fix here: https://github.com/hypothesis/h/pull/6312/commits/1dead57f7e48526a1c87527f33dee93a116e789e